### PR TITLE
Add people to about page

### PR DESCRIPTION
**Pull request by smcgregor**

### CODE REVIEW

File: site/gatsby-site/content/about.mdx [hunk 1]
```diff
@@ -49,32 +49,35 @@ The incident database is managed in a participatory manner by persons and organi
...
 
-
 # Collaborators
 
 **Open Source Contributors:** People that have contributed more than one pull request, graphics, site copy, or bug report to the AI Incident Database.
 
-* [César Varela](https://github.com/cesarvarela)
+* [César Varela](https://github.com/cesarvarela) is a Full Stack Engineer with the Responsible AI Collaborative
+* [Luna McNulty](https://lmcnulty.me/) is a UX Engineer with the Responsible AI Collaborative
 * [Alex Muscă](https://github.com/alexmcode)
```
Comment (lmcnulty) [inline site/gatsby-site/content/about.mdx:62]: This looks good so far as I'm concerned. I'll note that César https://cesarvarela.com and Alex https://alexmusca.com both have websites. Did they ask to list their GitHub profiles instead?

Comment (smcgregor) [reply site/gatsby-site/content/about.mdx:64]: I don't think so, but I'll let everyone determine their own link.

**Comment by smcgregor** [comment]
Added @cesarvarela to get this in staging (if he is good with the contents)

**Comment by smcgregor** [comment]
@cesarvarela I resolved the conflicts. I believe this is ready to go.

**Pull request merged**